internal/release/service.go

```diff
@@ -172,19 +189,6 @@ func (s *service) processFilters(ctx context.Context, filters []*domain.Filter,
 			}
 		}
 
-		// found matching filter, lets find the filter actions and attach
-		actions, err := s.actionSvc.FindByFilterID(ctx, f.ID)
-		if err != nil {
-			s.log.Error().Err(err).Msgf("release.Process: error finding actions for filter: %s", f.Name)
-			return err
-		}
-
-		// if no actions, continue to next filter
-		if len(actions) == 0 {
-			s.log.Warn().Msgf("release.Process: no actions found for filter '%s', trying next one..", f.Name)
-			return nil
```

Comment:
The `return nil`has to be a bug, right? I noticed with return nil, the log never shows the processing of the next filter in line, but if we continue instead, then it does.